test/evergreen.yml

```diff
@@ -5434,6 +5492,10 @@ buildvariants:
     - name: csuite-long-running
       batchtime: 1440 # once a day
     - name: live-restore-long
+    - name: live-restore-short-heavy-testing
```

Comment:
Instead of `heavy-testing` can we just use the postfix `-stress`?

Reply:
I was trying to avoid `stress` since we already use it for our other tests. How about `-extra-testing`?

Reply:
sgtm.